static/js/modules/columns.js

```diff
@@ -506,6 +506,26 @@ var reports = {
   }
 };
 
+var loans = [
+  {
+    data: 'committee',
+    orderable: false,
+    className: 'all column--large',
+    render: function (data, type, row) {
+      return row.committee.name;
+    }
+  },
+  {
+    data: 'loan_source_name',
+    orderable: false,
+    className: 'all column--large',
+  },
+  dateColumn({data: 'load_date', orderable: true, className: 'min-tablet hide-panel column--med'}),
```

Comment:
This is for the details panel, right? There are two dates we should probably show if there is room, `Date Incurred` and 'Date Due`.